posts/GSoC_2019_Project_First_Evaluation.md

```diff
@@ -0,0 +1,74 @@
+type: text
+author: Chao Xu
+---
+
+# GSoC 2019 PyCanChemAndYouCan2 (ChemCheck) First Evaluation
+
+  Thanks for the instructions from mentors Richard West([@rwest](https://github.com/rwest)), 
+Bryan Weber([@bryanwweber](https://github.com/bryanwweber)) and Kyle Niemeyer([@kyleniemeyer](https://github.com/kyleniemeyer)), the website
+is functioning after first month, and we will keep improving it in the rest of GSoC time. From what I mentioned in last post, ChemCheck is a
+web-based application for visualizing and diagnosing syntax and chemical errors in Chemkin and Cantera input files. In this case, we created basic 
+functions which allow users to convert their mechanism files to Cantera yaml input files easily (Cantera input file is going to be changed
+from cti file to yaml file, so we made some ajustments as well.).
+
+## Upload Page
+
+Users will upload their files to ChemCheck for checking, so we created an upload page which allows users to upload their mechanism, transport,
+thermo, and surface file. Here is the view of upload page.
+
+![image of upload page](https://github.com/12Chao/myproject/blob/master/images/upload%20page.png)
+
+## List Page
+
+  After files are uploaded, ChemCheck returns to a list page which lists all uploaded files in the app with id for each group of files.
+Users are able to either click on "Details" button to do further operation to their files or hit "Upload new mechanism" button to upload new files.
+
+![image of list page](https://github.com/12Chao/myproject/blob/master/images/list%20page.png)
+ 
+## Detail Page
+ 
+  In this page, users can either edit their files or hit "convert to YAML" button to convert files. The editing function is still in development.
+Also, there is a "Back to mechanism list" link for users going back to list view.
+
+![image of detail page](https://github.com/12Chao/myproject/blob/master/images/detail%20page.png)
+
+## Convert Page
+ 
+  If the file is converted successfully, a success message will be shown on the page.
+Users can see details and download the files from this page.
+
+![image of successfully uploaded page](https://github.com/12Chao/myproject/blob/master/images/convert%20succ.png)
+
+However, if the file is not converted successfully, the error message will be shown.
+
+![image of successfully uploaded page](https://github.com/12Chao/myproject/blob/master/images/convert%20fail.png)
+
+## Future Improvement 
+
+ChemCheck currently meets very basic requirements of our GSoC project, but there are bunch of things to be improved in the future.
+Here I make a list for improvements in next month:
+
+- Add function which allows users replace their files 
+- Report more detail about the error (like full stack trace and ck2yaml logging message) rather than just a error message.
+- Add Django login module, this function can help ChemCheck separate files because files uploaded by different users will saved under different
```

Comment:
```suggestion
- Add Django login module, this function can help ChemCheck separate files because files uploaded by different users will be saved under different
```